### PR TITLE
Added SharedFormulas. Expanded Part values.

### PR DESCRIPTION
**Pull request by ArcticWaffles**

### CODE REVIEW

File: BandNameGenerator/Genres/Country.cs [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+        private readonly List<string> singularNouns = new List<string>() { "Boot" };
         public List<string> SingularNouns => singularNouns;
 
-        private readonly static List<string> pluralNouns = new List<string>() { "da" };
+        private readonly static List<string> pluralNouns = new List<string>() { "Boots" };
         public List<string> PluralNouns => pluralNouns;
 
-        private readonly List<string> adjectives = new List<string>() { "da" };
+        private readonly List<string> adjectives = new List<string>() { "Dusty" };
         public List<string> Adjectives => adjectives;
```
Comment (JVimes) [inline BandNameGenerator/Genres/Country.cs]: Don't need explicit backing field on these (same for other genres). Just like:

```c#
public List<string> Adjectives { get; } = new List<string>() { "Dusty" };
```

That sets the initial value, backing field is implicit.

File: BandNameGenerator/SharedFormulas.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BandNameGenerator
+{
+    public static class SharedFormulas
```
Comment (JVimes) [inline BandNameGenerator/SharedFormulas.cs:8]: This is just fine. I'd also consider `PopularFormulas` as a name.

File: BandNameGenerator/Formula.cs [hunk 3]
```diff
@@ -13,15 +13,22 @@ public enum Part
...
+            OfThe,
+            MeA,
+            Dont
         }
 
         private readonly List<Part> parts;
 
         private readonly static List<string> numbers = new List<string>() { "3", "12", "100", "500", "A Thousand" };
 
+        private readonly static List<string> frequencies = new List<string>() { "Always", "Never", "Seldom" };
```
Comment (JVimes) [inline BandNameGenerator/Formula.cs:30]: Are these supposed to be private fields? As opposed to public, or properties.

File: BandNameGenerator/Formula.cs [hunk 4]
```diff
@@ -13,15 +13,22 @@ public enum Part
...
+
         public Formula(List<Part> parts)
```
Comment (JVimes) [inline BandNameGenerator/Formula.cs]: Unnecessary idea from me: You could replace `List<Part>`, here with an array and add the [params](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params) key word. Then it's as if the method takes an unlimited number of arguments, each of type `Part`. It avoids the caller having to do `new List<Formula.Part>(){...}` in cases like this:

```c#
numberPluralNoun = new Formula(new List<Formula.Part>(){ Formula.Part.Number, Formula.Part.PluralNoun });
```
Which could then be:
```c#
numberPluralNoun = new Formula(Formula.Part.Number, Formula.Part.PluralNoun);
```

**Review by JVimes — CHANGES_REQUESTED** [review]

File: BandNameGenerator/Genres/Country.cs (hunk 1 above)
Comment (ArcticWaffles) [reply BandNameGenerator/Genres/Country.cs]: Yep, don't mind me...just rusty from doing web dev for a year and a half. ;)

File: BandNameGenerator/Formula.cs (hunk 4 above)
Comment (ArcticWaffles) [reply BandNameGenerator/Formula.cs]: I like it!

**Review by JVimes — APPROVED** [review]

**Pull request merged**